modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,1073 @@
+= XDCR Conflict Logging
+:description: pass:q[XDCR detects and logs concurrent conflicts that occur in different clusters but on the same document version due to independent modifications by locally connected applications during Active-Active replication. You can manually review and resolve these conflicts to prevent data loss.]
```

Comment:
Please remove the last sentence -- "You can manually review and resolve these conflicts to prevent data loss."
Instead, say, "The conflict logs are for your information only.  The best practice in Active-Active systems is that application environments be designed to avoid conflicts."

The conflicts are always resolved by XDCR replication process -- like it's always done -- this feature does not change that.  So, we don't want to make people think otherwise.  Also, people can look at the conflict log information and do with it what they will -- the conflict logs are for their info only.